aip/0141.md

```diff
@@ -16,7 +16,9 @@ number of miles, number of nodes, etc.).
 ## Guidance
 
 Quantities with a clear unit of measurement (such as bytes, miles, and so on)
-**must** include the unit of measurement as the suffix.
+**must** include the unit of measurement as the suffix. When appropriate, units
+may use generally accepted abbreviations (for example, `distance_km` rather than
+`distance_kilometers`).
```

Comment:
We're going to have an AIP with kibi/kilo, IIRC? That'll be important for byte counts.

Reply:
Yep, that's https://github.com/googleapis/aip/issues/71